web/src/queries/storage/config-model.ts

```diff
@@ -298,6 +307,22 @@ export function usePartition(driveName: string, mountPath: string): PartitionHoo
   };
 }
 
+/**
+ * Hook for operating on the collection of drives from the model.
+ */
+export type DrivesHook = {
+  add: (driveName: string) => void;
+};
+
+export function useDrives(): DrivesHook {
```

Comment:
In the PR#1935 we added the `useModel` hook to query at model level, see https://github.com/agama-project/agama/pull/1935/commits/cb28ed6688f4d6b2c8716cd544d351dea36ca4a0#diff-c74d91a4c994f0b9fc0d945b55da1bb5d35a78503f1dfeb65c2b1b308a85dfc0R393. Maybe we can add `#addDrive` to that hook.

Reply:
I was looking for something like that, but I didn't find it (because is still not merged).